packages/contact-center/cc-components/src/components/StationLogin/station-login.tsx

```diff
@@ -95,81 +113,106 @@ const StationLoginComponent: React.FunctionComponent<StationLoginComponentProps>
       </dialog>
+      <dialog ref={ccSignOutModalRef} className="cc-logout-modal">
+        <Text tagname="h2" type="body-large-bold" className="modal-text">
+          Sign out of Contact Center
+        </Text>
+        <Text tagname="p" type="body-midsize-regular" className="modal-text">
+          Are you sure you want to sign out?
+        </Text>
+        <div className="cc-logout-modal-content">
+          <Button onClick={cancelClicked} variant="secondary" className="white-button">
+            Cancel
+          </Button>
+          <Button onClick={contactCenterLogout}>Sign out</Button>
+        </div>
+      </dialog>
       <div className="box station-login">
         <section className="section-box">
           <fieldset className="fieldset">
-            <Text tagName={'span'} type="heading-small-bold">
+            <Text tagname={'span'} type="heading-small-bold">
               Set your interaction preferences
             </Text>
           </fieldset>
           <fieldset className="fieldset">
-            <legend id="agent-login-label">Handle calls using</legend>
-            <div className="select-container">
-              <SelectNext
+            <legend id="agent-login-label">
+              Handle calls using
+              <Icon name="info-badge-filled" id="agent-login-info-badge" />
+            </legend>
+            <Tooltip
+              color="contrast"
+              id="agent-login-label-tooltip"
+              showArrow={true}
+              triggerID="agent-login-info-badge"
+            >
+              <Text tagname={'div'} type="body-large-regular" className="agent-login-popover">
+                This is your preferred method for receiving and making calls. Choose between your phone number,
+                extension (if available), or your web browser.
+              </Text>
+            </Tooltip>
+            <div className="">
+              <Select
                 id="login-option"
-                direction="bottom"
-                showBorder
-                aria-labelledby="agent-login-label"
-                items={loginOptions.map((name, id) => ({key: id.toString(), name}))}
-                selectedKey={agentLoginValue}
-                onSelectionChange={selectLoginOption}
+                name="login-option"
+                onChange={(event: CustomEvent) => selectLoginOption(event.detail.value)}
                 className="station-login-select"
               >
-                {(item) => (
-                  <Item textValue={item.name} key={item.key}>
-                    <Text className="state-name" tagName={'small'}>
-                      {item.name}
-                    </Text>
-                  </Item>
-                )}
-              </SelectNext>
-              <Icon className="select-arrow-icon" name="arrow-down-bold" title="" />
+                {loginOptions.map((option, index) => {
+                  return (
+                    <Option key={index} value={option}>
```

Comment:
If it is the re-login, then the deviceType value should be selected value. At the moment with this change, it just resets to 0th value all the time I believe. 

The value should come from the store to the props. Please check the `selectedKey` in the deleted change